File: frontend-svelte/src/lib/api/models/Component.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Label } from './Label';
import type { Translation } from './Translation';

export type Component = {
    id?: string;
    version?: number;
    name?: string | null;
    createdAt?: string;
    updatedAt?: string;
    labels?: Array<Label> | null;
    translations?: Array<Translation> | null;
};

File: src/desktop/Models/Component.cs
using System;
using System.Collections.Generic;

namespace Transleet.Desktop.Models;


public class Component
{
    public Guid Id { get; set; }
    public string? Name { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
    public DateTimeOffset UpdatedAt { get; set; }
    public ICollection<Label>? Labels { get; set; }
    public ICollection<Translation>? Translations { get; set; }
}
